src/main/java/aztech/modern_industrialization/machines/components/CasingComponent.java

```diff
@@ -42,15 +43,15 @@
 import net.minecraft.world.level.block.Blocks;
 import org.jetbrains.annotations.Nullable;
 
-public class CasingComponent implements IComponent, DropableComponent {
+public class CasingComponent implements IComponent, DropableComponent, CableTierHolder {
```

Comment:
This is not how the holder is meant to be used IMO. A check for `blockEntity instanceof CableTierHolder` will never succeed like this.

Reply:
What do you think about adding the holder interface to single block electric machines? And then what about multiblocks? How it would work for multiblocks is a bit ambiguous. I think keeping the getCasingTier method for the CasingComponent is good even if the interface is removed from it.

Reply:
Yes we can add it to electric machines. Multiblocks can probably wait until someone requests them. Yes to keep the method without the interface.